build/Targets/VSL.Versions.targets

```diff
@@ -32,9 +32,11 @@
     <MicrosoftVisualStudioShell15Version>15.0.26507-alpha</MicrosoftVisualStudioShell15Version>
     <MicrosoftVisualStudioShellDesign>15.0.26507-alpha</MicrosoftVisualStudioShellDesign>
     <MicrosoftVisualStudioManagedInterfaces>8.0.50727</MicrosoftVisualStudioManagedInterfaces>
+    <!-- NOTE: All three of these should be updated at once to reference the same roslyn build-->
```

Comment:
If they all have to be updated at the same time, can it be one property? Or atleast the value of these property can be one property whose value is the 2.3.0-beta3...